docs/developer/updating-otel/README.md

```diff
@@ -95,6 +95,18 @@ Unfortunately, updating Otel dependencies is not straightforward:
 You can refer to [PR grafana/agent#5290](https://github.com/grafana/agent/pull/5290)
 for an example on how to update Alloy.
 
+### Notify community component contributors
+
+You can find the community contributor GitHub handles at the top of the community component files (the components that have "Community" set to true).
+Once the PR is created, you can ping them in the PR and message them on Slack. Each contributor should create a PR in your branch with the updated component.
+Notify the contributors a few days before the release so they have enough time to do the update.
```

Comment:
Most of the time when you read these docs you will be doing the release, so I'm not sure if it's the right place for a reminder like this.

Reply:
this is the "updating-otel" doc. The update should be done prior to the release and should be started at least one week before cutting the RC